### PR TITLE
Add icons and gutter to post

### PR DESCRIPTION
**Pull request by slocker3**
- Add gutter to post
- Add icons to `meta-head`s
- Changed `fill` color on `topic.svg` and `people.svg` to `#a60f2d`
- updated content in `post.html` styleguide

### CODE REVIEW

File: css/01-layout.css [hunk 1]
```diff
@@ -186,22 +186,37 @@ footer.site-footer {
 	padding: 4rem 0 !important;
 }
 
+.article-body.side-right .one {
+	padding-right: 2rem !important;
```
Comment (jeremyfelt) [inline css/01-layout.css:190]: 🙈

Comment (slocker3) [reply css/01-layout.css:190]: 👍  !important !!!

**Review by jeremyfelt — APPROVED** [review]
👍

File: css/01-layout.css (hunk 1 above)
Comment (slocker3) [reply css/01-layout.css:190]: I will make a note to look for another solution to the `!important` tags in a future push.

Comment (philcable) [reply css/01-layout.css:190]: `#jacket #binder` all the way!

**Pull request merged**